Sources/YOLO/YOLOView.swift

```diff
@@ -788,7 +788,7 @@ public class YOLOView: UIView, VideoCaptureDelegate {
       let width = bounds.width
       let height = bounds.height
 
-      let topMargin: CGFloat = 0
+      let topMargin: CGFloat = height * 0.05  // Increased top margin
```

Comment:
@asabri97 is this change supposed to be part of this PR? Also see change below same file on L879 please.